zebedee-reader/src/main/java/com/github/onsdigital/zebedee/reader/api/ReadRequestHandler.java

```diff
@@ -43,6 +48,7 @@ private interface ResourceFinder<R> {
 
     private final static String LATEST = "latest";
     private ZebedeeReader reader;
+    private Tracer tracer = GlobalOpenTelemetry.getTracer("zebedee", "");
```

Comment:
This should probably be "zebedee-reader" rather than "zebedee" as this sits within the zebedee-reader deployment